pom.xml

```diff
@@ -90,24 +87,35 @@
             <name>Local repository for hosting jars not available from network repositories.</name>
             <url>file://${project.basedir}/local_lib</url>
         </repository>
+        <repository>
+            <id>payara-patched-externals</id>
```

Comment:
Does this further tie us to Payara? Will it be harder to move to Liberty or Wildfly?

Reply:
Yes. But we are pretty bound to Pasta anyway, as we rely on their patched lib releases. As long as those don't hit upstream, it would lead to troubles when deploying to Wildfly/Liberty/... anyway.